workos/sso.py

```diff
@@ -86,6 +87,25 @@ def get_authorization_url(
 
         return prepared_request.url
 
+    def get_profile(self, accessToken):
+        """
+        Verify that SSO has been completed successfully and retrieve the identity of the user.
+
+        Args:
+            accessToken (str): the token used to authenticate the API call
+
+        Returns:
+            WorkOSProfile
+        """
+
+        params = {"access_token": accessToken}
```

Comment:
I don't know that `params` is going to do the right thing here. I think this will add an `access_token` query parameter, but we need to set the access token in the `Authorzation: Bearer <ACCESS_TOKEN>` header.

Reply:
Thanks @maxdeviant  - I see what you're saying. The `request` function has an option for a token

```
if token:
            headers["Authorization"] = "Bearer {}".format(token)
```

 so I will pass the token in there and test the behavior.